pkg/misc/utils.go

```diff
@@ -41,6 +41,13 @@ const (
 	RabbitMQDefaultUserCredentialsPasswordKey = "admin-password"
 )
 
+const (
```

Comment:
Two things:

1) Join the two `const ()` as keeping them separate will just break the code checks
2) The values don't really make sense. Here you're defining the **key** of the secret, not the default username/password value. So they would be something like `username` and `password`, respectively.